app/main.py

```diff
@@ -1,3 +1,64 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
```

Comment:
Consider adding type validation in the `__init__` method to ensure `km` is either an `int` or `float`. This will prevent potential errors when creating a `Distance` object with an invalid type.